src/qbittorrent.exe.manifest

```diff
@@ -37,9 +37,10 @@
   </compatibility>
 
   <!-- Enable long paths that exceed MAX_PATH in length -->
-  <asmv3:application>
-    <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">
```

Comment:
I believe you can just change it to `http://schemas.microsoft.com/SMI/2020/WindowsSettings`